cloudslang-entities/src/main/java/io/cloudslang/lang/entities/LoopStatement.java

```diff
@@ -37,4 +40,20 @@ protected LoopStatement() {
     public String getExpression() {
         return expression;
     }
+
```

Comment:
I prefer these methods over doing it manually, since
- we don't need to define each method in each subclass
- reduced code size
- standard output format
- can be considered best practice, better control over data

I like this approach, but we can go with explicit methods for each subclass - WDYT?